### PR TITLE
Button when hovering ghost page added

### LINKED ISSUE #2847: Double click is needed to create a page translation
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | no
| Sulu Version | 1.3.0
| Browser Version | Tested in Firefox and Chrome

#### Actual Behavior
A double click is needed to create the page "Test" in en.
double-click

#### Expected Behavior

A single click should be enough.

#### Steps to Reproduce

1. Create a page in "de".
2. Switch to "en" in the content tree view.
3. Try to add a translation for the page.

### PR DESCRIPTION
**Pull request by ReneZumtobel**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #2847 
| Related issues/PRs | https://github.com/massiveart/husky/pull/722
| License | MIT
| Documentation PR | sulu/sulu-docs#prnum

#### What's in this PR?

Fixed create hover button for ghost pages.

### CODE REVIEW

**Comment by ReneZumtobel** [comment]
@danrot

File: src/Sulu/Bundle/ContentBundle/Resources/translations/sulu/backend.nl.xlf [hunk 1]
```diff
@@ -747,6 +747,7 @@
                 <source>labels.error.remove-draft-desc</source>
                 <target>Het concept kon niet verwijderd worden.</target>
             </trans-unit>
+<<<<<<< fbfdaa758be2ec5dcc26dd26f00931c702c686d0
```
Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Resources/translations/sulu/backend.nl.xlf]: Please remove

File: src/Sulu/Bundle/ContentBundle/Resources/translations/sulu/backend.fr.xlf [hunk 2]
```diff
@@ -751,6 +751,7 @@
                 <source>sulu-content.teaser.content</source>
                 <target>Pages</target>
             </trans-unit>
+<<<<<<< fbfdaa758be2ec5dcc26dd26f00931c702c686d0
```
Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Resources/translations/sulu/backend.fr.xlf]: Please remove

File: src/Sulu/Bundle/ContentBundle/Resources/translations/sulu/backend.en.xlf [hunk 3]
```diff
@@ -751,6 +751,7 @@
                 <source>sulu-content.teaser.content</source>
                 <target>Pages</target>
             </trans-unit>
+<<<<<<< fbfdaa758be2ec5dcc26dd26f00931c702c686d0
```
Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Resources/translations/sulu/backend.en.xlf]: Please remove

File: src/Sulu/Bundle/ContentBundle/Resources/translations/sulu/backend.de.xlf [hunk 4]
```diff
@@ -751,6 +751,7 @@
                 <source>sulu-content.teaser.content</source>
                 <target>Seiten</target>
             </trans-unit>
+<<<<<<< fbfdaa758be2ec5dcc26dd26f00931c702c686d0
```
Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Resources/translations/sulu/backend.de.xlf]: Please remove

**Pull request merged**